resources/views/show.blade.php

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Show</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+</head>
+
+<body>
+    <a href="/" class="btn btn-dark" style= "margin:30px">戻る</a>
+    <h1 class ="text-center">投稿のリスト</h1>
+    <a style="margin-left:120px" href="/add" class="btn btn-primary">足す</a>
+    <br>
+    <br>
+    <div class="row" style="margin:100px">
+        <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+            <div class="well blog">
+                 
+                <div class="row">
+                    <div class="col-xs-5 col-sm-5 col-md-12 col-lg-12">   
+                    </div>
+                    <div class="col-xs-12 col-sm-4 col-md-4 col-lg-12">
+                        <div class="blog-details">
+                        @foreach($posts as $post)
+                            <tr>
+                                <h2> {{$post->title}}</h2>
+                                <p> {{$post->body}}</p>
+                                <td><a href="/delete/{{$post->id}}" class="btn btn-danger">削除</a></td>
```

Comment:
```suggestion
                                <td><a href="{{ route('post.destroy', $post->id) }}" class="btn btn-danger">削除</a></td>
```
Dùng helper route() như trên, ko fix cứng như vậy nhé. Bổ sung localization